plugins/kotlin/jvm-debugger/test/testData/stepping/packets/evaluatableGetters.out

```diff
@@ -0,0 +1,2348 @@
+        JavaValue[getter] x10: int (evaluatableGetters.kt:13)
+        JavaValue[getter] x9: int (evaluatableGetters.kt:12)
+        JavaValue[getter] x8: int (evaluatableGetters.kt:11)
+        JavaValue[getter] x7: int (evaluatableGetters.kt:10)
+        JavaValue[getter] x6: int (evaluatableGetters.kt:9)
+        JavaValue[getter] x5: int (evaluatableGetters.kt:8)
+        JavaValue[getter] x4: int (evaluatableGetters.kt:7)
+        JavaValue[getter] x3: int (evaluatableGetters.kt:6)
+        JavaValue[getter] x2: int (evaluatableGetters.kt:5)
+        JavaValue[getter] x1: int (evaluatableGetters.kt:4)
+evaluatableGetters.kt:116
+Packets sent: 104, methods called: 7
+KotlinStackFrame (evaluatableGetters.kt:116)
+    JavaValue[local] x10: evaluatableGetters.X10 = evaluatableGetters.X10@hashCode (evaluatableGetters.kt:114)
+        DummyMessageValueNode = No fields to display
+        JavaValue[getter] x10: int (evaluatableGetters.kt:13)
+        JavaValue[getter] x9: int (evaluatableGetters.kt:12)
+        JavaValue[getter] x8: int (evaluatableGetters.kt:11)
+        JavaValue[getter] x7: int (evaluatableGetters.kt:10)
+        JavaValue[getter] x6: int (evaluatableGetters.kt:9)
+        JavaValue[getter] x5: int (evaluatableGetters.kt:8)
+        JavaValue[getter] x4: int (evaluatableGetters.kt:7)
+        JavaValue[getter] x3: int (evaluatableGetters.kt:6)
+        JavaValue[getter] x2: int (evaluatableGetters.kt:5)
+        JavaValue[getter] x1: int (evaluatableGetters.kt:4)
+    JavaValue[local] x20: evaluatableGetters.X20 = evaluatableGetters.X20@hashCode (evaluatableGetters.kt:115)
+        DummyMessageValueNode = No fields to display
+        JavaValue[getter] x20: int (evaluatableGetters.kt:23)
+        JavaValue[getter] x19: int (evaluatableGetters.kt:22)
+        JavaValue[getter] x18: int (evaluatableGetters.kt:21)
+        JavaValue[getter] x17: int (evaluatableGetters.kt:20)
+        JavaValue[getter] x16: int (evaluatableGetters.kt:19)
+        JavaValue[getter] x15: int (evaluatableGetters.kt:18)
+        JavaValue[getter] x14: int (evaluatableGetters.kt:17)
+        JavaValue[getter] x13: int (evaluatableGetters.kt:16)
+        JavaValue[getter] x12: int (evaluatableGetters.kt:15)
+        JavaValue[getter] x11: int (evaluatableGetters.kt:14)
+        JavaValue[getter] x10: int (evaluatableGetters.kt:13)
+        JavaValue[getter] x9: int (evaluatableGetters.kt:12)
+        JavaValue[getter] x8: int (evaluatableGetters.kt:11)
+        JavaValue[getter] x7: int (evaluatableGetters.kt:10)
+        JavaValue[getter] x6: int (evaluatableGetters.kt:9)
+        JavaValue[getter] x5: int (evaluatableGetters.kt:8)
+        JavaValue[getter] x4: int (evaluatableGetters.kt:7)
+        JavaValue[getter] x3: int (evaluatableGetters.kt:6)
+        JavaValue[getter] x2: int (evaluatableGetters.kt:5)
+        JavaValue[getter] x1: int (evaluatableGetters.kt:4)
+evaluatableGetters.kt:117
+Packets sent: 108, methods called: 8
```

Comment:
After the changes, the number of method calls is increased, which is expected. Could you please add a test that will show that there are no additional method calls if the metadata is cached?